src/components/CourseTable/index.jsx

```diff
@@ -133,14 +133,7 @@ class CourseTable extends React.Component {
     return (
       <>
         <div className="row">
-          <div className="col-2 float-left">
-            <ButtonToolbar className="mb-3" leftJustify>
-              <Link to="/courses/new">
-                <button type="button" className="btn btn-primary">New Course</button>
-              </Link>
-            </ButtonToolbar>
-          </div>
-          <div className="col-5 float-right">
+          <div className="px-3 float-left" style={{ width: '44%' }}>
```

Comment:
have you checked the impact of this %age on different screen sizes? I would advocate using bootstrap classes for managing widths if possible.

Reply:
if we only use bootstrap classes, I don't think we can remove the whitespace between new course button and search bar...note that this empty space was present before too (only it was placed between course button and filter bar earlier) More discussion about bootstrap's limitations in this regard are [here](https://stackoverflow.com/questions/28750907/how-to-customize-bootstrap-column-widths)

On smaller screens the components start overlapping but that is not due to this but rather because the new course button does not respect the width of its container (this can also be seen on master)